[2024-12-18T20:10:35.343+0000] {__init__.py:54} DEBUG - Loading core task runner: StandardTaskRunner
[2024-12-18T20:10:35.369+0000] {base_task_runner.py:73} DEBUG - Planning to run as the  user
[2024-12-18T20:10:35.380+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-18T20:10:35.397+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:10:35.398+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-12-18T20:10:35.399+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-12-18T20:10:35.400+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-12-18T20:10:35.409+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:10:35.410+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]>
[2024-12-18T20:10:35.416+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-12-18T20:10:35.418+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:10:35.421+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-12-18T20:10:35.425+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-12-18T20:10:35.426+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:10:35.427+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [queued]>
[2024-12-18T20:10:35.427+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-18T20:10:35.448+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_usernames> on 2024-12-18 20:10:32.326443+00:00
[2024-12-18T20:10:35.460+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2784) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-18T20:10:35.462+0000] {standard_task_runner.py:72} INFO - Started process 2789 to run task
[2024-12-18T20:10:35.461+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tiktok_dag', 'load_usernames', 'manual__2024-12-18T20:10:32.326443+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/tiktok_dag.py', '--cfg-path', '/tmp/tmp1eoauxf2']
[2024-12-18T20:10:35.464+0000] {standard_task_runner.py:105} INFO - Job 84: Subtask load_usernames
[2024-12-18T20:10:35.465+0000] {cli_action_loggers.py:79} DEBUG - Calling callbacks: [<function default_action_log at 0x7f2a69185da0>]
[2024-12-18T20:10:35.524+0000] {task_command.py:467} INFO - Running <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:10:32.326443+00:00 [running]> on host 00d1ef865b53
[2024-12-18T20:10:35.577+0000] {taskinstance.py:2384} DEBUG - Clearing XCom data
[2024-12-18T20:10:35.593+0000] {retries.py:95} DEBUG - Running RenderedTaskInstanceFields.write with retries. Try 1 of 3
[2024-12-18T20:10:35.619+0000] {retries.py:95} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-12-18T20:10:35.631+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tiktok_dag' AIRFLOW_CTX_TASK_ID='load_usernames' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T20:10:32.326443+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T20:10:32.326443+00:00'
[2024-12-18T20:10:35.632+0000] {__init__.py:115} DEBUG - Preparing lineage inlets and outlets
[2024-12-18T20:10:35.633+0000] {__init__.py:154} DEBUG - inlets: [], outlets: []
[2024-12-18T20:10:35.634+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-18T20:10:35.662+0000] {tiktok_dag.py:43} INFO - Successfully loaded 3 usernames from /opt/***/dags/influencers.csv
[2024-12-18T20:10:35.662+0000] {tiktok_dag.py:44} INFO - Usernames loaded: ['mrbeast', 'jadeanna', 'feyenoord']
[2024-12-18T20:10:35.683+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-18T20:10:35.684+0000] {__init__.py:77} DEBUG - Lineage called with inlets: [], outlets: []
[2024-12-18T20:10:35.697+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-18T20:10:35.698+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tiktok_dag, task_id=load_usernames, run_id=manual__2024-12-18T20:10:32.326443+00:00, execution_date=20241218T201032, start_date=20241218T201035, end_date=20241218T201035
[2024-12-18T20:10:35.712+0000] {cli_action_loggers.py:98} DEBUG - Calling callbacks: []
[2024-12-18T20:10:35.759+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-18T20:10:35.806+0000] {dagrun.py:968} DEBUG - number of tis tasks for <DagRun tiktok_dag @ 2024-12-18 20:10:32.326443+00:00: manual__2024-12-18T20:10:32.326443+00:00, state:running, queued_at: 2024-12-18 20:10:32.430647+00:00. externally triggered: True>: 2 task(s)
[2024-12-18T20:10:35.807+0000] {dagrun.py:989} DEBUG - number of scheduleable tasks for <DagRun tiktok_dag @ 2024-12-18 20:10:32.326443+00:00: manual__2024-12-18T20:10:32.326443+00:00, state:running, queued_at: 2024-12-18 20:10:32.430647+00:00. externally triggered: True>: 1 task(s)
[2024-12-18T20:10:35.808+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:10:32.326443+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:10:35.809+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:10:32.326443+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:10:35.811+0000] {taskinstance.py:2613} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:10:32.326443+00:00 [None]>
[2024-12-18T20:10:35.819+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-18T20:10:35.831+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
